content/en/meet.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Meeting the gRPC Maintainers
+linkTitle: Meet
+menu:
+  main: {weight: 1}
+---
+
+{{< blocks/cover image_anchor="top" height="sm" color="primary" >}}
+{{< page/header >}}
+{{< /blocks/cover >}}
+
+<div class="container l-container--padded">
+
+<div class="row">
+{{< page/toc collapsed=true placement="inline" >}}
+</div>
+
+<div class="row">
+<div class="col-12 col-lg-8">
+
+Dear gRPC users,
+
+Here are the gRPC maintainers. We're trying to better understand the needs of
+gRPC users to help plan and prioritize future efforts. We are hoping you'd be
+willing to do a short call to tell us about your developer experience with gRPC,
+with observability, and with microservices – about what we're getting right, but
+more importantly about where things could be better.
+
+To schedule a 30 minutes session, please fill out <a name="this-survey" href="https://docs.google.com/forms/d/e/1FAIpQLSe1klQIom5SnpL7czmNFI9MZHy_eNwOCHghV0e61hTzY93qWw/viewform?usp=sf_link">this survey</a>.
```

Comment:
```suggestion
To schedule a 30 minute session, please fill out <a name="this-survey" href="https://docs.google.com/forms/d/e/1FAIpQLSe1klQIom5SnpL7czmNFI9MZHy_eNwOCHghV0e61hTzY93qWw/viewform?usp=sf_link">this survey</a>.
```

Reply:
TIL if we are using "X minute" as adjective, we can skip the "s" in the end.

Reply:
It's actually obligatory for nominal modifiers in general, hence "dog houses", not "dogs houses". Though certain irregularly pluralized nouns are in free variation on this, so some people will say "women engineers" while others will say "woman engineers."